modules/costs/app/helpers/timer_helper.rb

```diff
@@ -0,0 +1,47 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2023 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module TimerHelper
+  def pad(val)
+    time = '00'
+    unless val.nil?
+      time = val > 9 ? val.to_s : "0#{val}"
+    end
+    time
+  end
+
+  def format_time_entry_as_elapsed(started_at)
+    seconds = Time.zone.now - started_at
+    parts = ActiveSupport::Duration
+      .build(seconds)
+      .parts
+      .transform_values(&:round)
+
+    "#{pad(parts[:hours])}:#{pad(parts[:minutes])}:#{pad(parts[:seconds])}"
```

Comment:
We can achieve the same formatting without a custom padding method.
```suggestion
    "%02d:%02d:%02d" % [parts.fetch(:hours, 0), parts.fetch(:minutes, 0), parts.fetch(:seconds, 0)]
```
This will also output  `#=> "01:00:00"` without using the pad method.